2001-3000/LIT2192Profit.xml

```diff
@@ -4,9 +4,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Profitable Compiler. Abridgement of the Explication of the Truth of the
-               Faith</title>
-            <author>Giyorgis Wäldä ʿAmid</author>
+            <title xml:id="t1">Abridgement of the Explication of the Truth of the Faith</title>
```

Comment:
Perhaps "An abridged explication ..."?

Reply:
As I understand it, maybe a better way of clarifying the formulation would be to write "Abridgement of the work known as 'Explication of the Truth of the Faith'" or similar